scipy/special/tests/test_basic.py

```diff
@@ -4105,47 +4140,51 @@ def test_big_numbers(self):
         ans = asarray([48063331393110, 48004081105038305])
         n = [25, 30]
         k = [17, 4]
-        assert array_equal(stirling2(n, k), ans)
+        assert array_equal(stirling2(n, k, exact=True), ans)
```

Comment:
I don't see any tests where the asymptotic expansion will be used. Just set the rtol to like `1e-4` or something.

Reply:
I added some `for n in range(51, 101, 5):` tests in the latest commit `
2fe884e`